release-team/release-team-removal.md

```diff
@@ -0,0 +1,52 @@
+# Release Team Removal Process
+	
+The release team is reassembled with a new team for each cycle. All team members are automatically removed if they do not re-apply and are selected for the next release cycle. While this process generally functions smoothly, there are instances when a team member might leave mid-cycle or the release team leaders may decide to part ways early. 
+
+## Resignation
+Voluntary resignation is outlined in the [shadow handbook](https://github.com/leonardpahlke/sig-release/blob/master/release-team/shadows.md). Role lead should discuss with Release Lead and Emeritus Advisor (EA) to find a replacement if they would like to step down.
+
+
+Remaining on the team without contributing creates an unhealthy environment for the rest of the team and negatively affects the outcome of the release. We strive to be inclusive and empathetic to individuals' circumstances.
+The following outline is not meant to be definitive or punitive but rather outlines general guidance for leads on the process.  
+## Shadow Removal Guideline
+
+Reasons for removal could be including but not limited to:
+- Failure to attend shadow orientation hosted by the EA (we host 2-3 sessions in different timezones)
+- Failure to attend shadow orientation hosted by the role lead (if lead doesn't offer timezone-appropriate meeting, shadow should communicate to lead about obtaining and watching the recording)
```

Comment:
```suggestion
- Failure to attend the orientation hosted by the Role Lead (if the Role Lead doesn't offer a timezone-appropriate meeting, the shadow should communicate to the Role Lead about obtaining the provided information async and watching the recording)
```

We have the shadow orientation by the EA. I would not name team sync’s shadow orientation meetings as it might confuse folks.